lib/buildmodel.js

```diff
@@ -35,14 +35,10 @@ function fetchPipelineId(datastore, jobId, callback) {
  *                                        keys: "pipelineID" and "scmUrl"
  */
 function fetchScmUrl(datastore, params, callback) {
```

Comment:
Not sure where else would use it, but I can add it to the pipeline Model since it would make more sense staying there.